### DISCUSSION TITLE
Stored Procedures in Phalcon

### DISCUSSION THREAD

**Discussion by Mosin0Ali** [phalcon/cphalcon#15914] [Q&A]
How to call stored procedures written in My SQL  in Phalcon?
When I try to execute my stored procedure like:
ControllerBase::getInstance("CALL mayorToDistrict($md->party_id,$partyName,$md->district_id,$districtName,$md->candidate_type_id,$md->win,$md->leaded)")->execute();
It throws an error
Caught Excpetion:Syntax error, unexpected token IDENTIFIER(CALL), near to ' mayorToDistrict(3,'एमाले /CPN-UML',62,'डोल्पा / Dolpa',1,4,0)', when parsing: CALL mayorToDistrict(3,'एमाले /CPN-UML',62,'डोल्पा / Dolpa',1,4,0) (88)
It seems phalcon is not able to recognize CALL 
I even tried wrapping it in quotes like:
$sql="CALL `mayorToDistrict`('$md->party_id','$partyName','$md->district_id','$districtName','$md->candidate_type_id','$md->win','$md->leaded');";
and then calling it like
ControllerBase::getInstance($sql)->execute(); 
but it throws the same exception.

**Comment by niden** [comment]
Phalcon does not have native support for stored procedures. You could use the custom dialect feature as described here:

https://docs.phalcon.io/4.0/en/db-layer#custom-1
